### PR TITLE
Update the CS fixer to version 2

### PR DESCRIPTION
**Pull request by leofeyer**
Version 2 now supports (aka requires) `use` statements in DCA files as discussed [here](https://github.com/contao/contao/pull/1810#discussion_r443579403).

### CODE REVIEW

**Comment by m-vo** [comment]
I really like the shorter notation, makes it much more readable :+1: .

I have a hard time to find the actual code changes (not those performed by the cs fixer) in the vast amount of total changes, though. I'd really love to see several (non-squashed) commits for these kind of things even after merging like with the semi-linear merge history strategy.

File: news-bundle/src/Resources/contao/dca/tl_news_feed.php [hunk 1]
```diff
@@ -440,14 +455,14 @@ public function generateFeed()
 			return;
 		}
 
-		$this->import('Contao\News', 'News');
+		$this->import('News', 'News');
```
Comment (fritzmg) [inline news-bundle/src/Resources/contao/dca/tl_news_feed.php:458]: Shouldn't this be changed to

```php
$this->import('News')
````
?

Also.. does this actually work? May be it needs to be

```php
$this->import(News::class, 'News')
```

? (I never use `$this->import` myself, so I am just unsure about it)

Comment (leofeyer) [reply news-bundle/src/Resources/contao/dca/tl_news_feed.php:458]: You are right, it seems that I have missed some occurrences. Fixed in #1863.

**Comment by leofeyer** [comment]
> I have a hard time to find the actual code changes (not those performed by the cs fixer) in the vast amount of total changes, though.

That is probably because there are none. 😄 All changes in this PR are CS fixes.

**Comment by m-vo** [comment]
I see some in `monorepo.yml` and `composer.json`. :wink:

**Pull request merged**